Domains/1-Common/DocumentMetadata/DocumentMetadata.ecschema.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="DocumentMetadata" alias ="dmd" version="01.00.00" description="Metadata from a Document Management System" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.14" alias="bis"/>
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>

```

Comment:
Do we want to remove this custom attribute before publishing

Reply:
This PR puts the schema in master as the "WIP" copy. We expect all WIP copies to claim they are not ready for production.

As part of the second PR releasing this schema, we will change that flag to "production-ready" on the released copy. The WIP copy will remain as "not ready for production".